<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogo</class>
 <widget class="QMainWindow" name="Dialogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialogo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 43);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOcupar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLiberar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtocupante">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>Matricula</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
